index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>theyalow</title>
+    <link rel="stylesheet" href="./style/style.css">
+
+</head>
+
+<body>
+    <div class="wrapper">
+        <main>
+            <div class="container">
+                <header class="first-line">
+                    <div class="logo">
+                        <a href="#"><img src="./img/logo.png" alt="Main logo"></a>
+                        <h1 class="logo-text"><a href="#">theyalow</a></h1>
+                    </div>
+                    <nav>
+                        <ul>
+                            <li><a href="#" target="_blank">About us</a></li>
+                            <li><a href="#" target="_blank">Services</a></li>
+                            <li><a href="#" target="_blank">Contact</a></li>
+                        </ul>
+                    </nav>
+                </header>
+                <article class="second-line">
+                    <video controls poster="./img/poster.png" preload="none">
+                        <source src="#" type="video/mp4">
+                    </video>
+                    <h2><span>The cool</span><br>the minimal</h2>
```

Comment:
Cool that you've made use of "h" elements in the right order :) h1 element can't be placed in the place more than one time